manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.locale.en-US.yaml

```
@@ -1,13 +1,24 @@
-# Created using wingetcreate
+# Created using wingetcreate, modified manually
 
 PackageIdentifier: Badlion.BadlionClient
 PackageVersion: 3.2.0
 PackageLocale: en-US
 Publisher: Badlion
+#PublisherUrl: https://www.badlion.net
+#PublisherSupportUrl: https://support.badlion.net/
+#PrivacyUrl: https://www.badlion.net/privacy
+# Author:
 PackageName: Badlion Client
-License: © 2013-2021 ESL Gaming Online, Inc.
-# LicenseUrl: https://www.badlion.net/tos
-# PrivacyUrl: https://www.badlion.net/privacy
-ShortDescription: Badlion Client is a Minecraft launcher with lots of features.
+#PackageUrl: https://client.badlion.net/
+License: Proprietary
+#LicenseUrl: https://www.badlion.net/tos
+Copyright: © 2013-2021 ESL Gaming Online, Inc.
+# CopyrightUrl: 
+ShortDescription: Badlion Client Minecraft Launcher
+Description: Badlion Client is a Minecraft launcher with many added features. It includes anticheat, built-in FPS boost and other mods, and is compatible with most versions of minecraft 1.7 or newer.
+Moniker: BLC
+Tags:
+- minecraft
+- optifine
 ManifestType: defaultLocale
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```

```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.